fixup-mountpoints

```diff
@@ -207,6 +207,33 @@ case "$DEVICE" in
             -e 's block/platform/msm_sdcc.1/by-name/userdata mmcblk0p39 ' \
```

Comment:
Please rename this commit as per naming conventions of other commits

m7spr mountpoints are not the same numbers-wise:) (they are listed in the fixups already)